### DISCUSSION TITLE
Will the osu stable's failing effect / fail screen be brought into lazer?

### DISCUSSION THREAD

**Discussion by finesseandstyle** [ppy/osu#16598] [Q&A]
I don't like the current failing effect in lazer. The sound used is sudden and the red screen is annoying as well as the screen that moves leaves a black background.

I hope it's just a placeholder cause the original is much more satisfying and smoother for just being a fail state.

**Comment by bdach** [comment]
the fail sound effect has been changed in the 2022.118.0 release. did you try that one?

the stable effect may eventually be brought back with the rest of gameplay UI skinning but it's not necessarily a high priority item.

**Reply by finesseandstyle** [reply]
I tried the latest release and my opinion maintains. I just think it has less charm over the original which makes failing less frustrating with the funny distorted music and the falling notes. Thanks for the quick answer tho
